umap/static/umap/js/modules/caption.js

```diff
@@ -7,6 +7,15 @@ export default class Caption {
     this.map = map
   }
 
+  isOpen() {
+    return !!document.querySelector('.on .umap-caption')
+  }
+
+  redraw() {
+    if (!this.isOpen()) return
+    this.open()
+  }
+
   open() {
```

Comment:
Would it make more sense to call it `draw()`?

Reply:
You mean because of the `redraw` method ? In this case I'd better change this method name in `reopen` ?